ftw/zipexport/generation.py

```diff
@@ -33,6 +34,17 @@ def __exit__(self, exc_type, exc_value, traceback):
         self.zip_file.__exit__(exc_type, exc_value, traceback)
         self.tmp_file.__exit__(exc_type, exc_value, traceback)
 
+    def add_folder(self, folder_path):
```

Comment:
I was thinking about this, too. But I decided to use `add_folder` because we do exactly that. 

Like on a filesystem, you don't create explicitly an empty folder. You just create a folder and this new folder is empty because it's new 😉 

@jone What do you prefer?